src/Cms/Page.php

```diff
@@ -1484,24 +1484,6 @@ public function dragText(string $type = null): string
         return $this->panel()->dragText($type);
     }
 
-    /**
-     * Returns the panel icon definition
-     * according to the blueprint settings
-     *
-     * @deprecated 3.6.0 Use `->panel()->icon()` instead
-     * @todo Add `deprecated()` helper warning in 3.7.0
-     * @todo Remove in 3.8.0
-     *
-     * @internal
-     * @param array|null $params
-     * @return array
-     * @codeCoverageIgnore
-     */
-    public function panelIcon(array $params = null): array
-    {
-        return $this->panel()->icon($params);
-    }
-
```

Comment:
Can we completely remove this method without prior deprecation?

Reply:
The deprecated method is still inherited from `ModelWithConent`